src/Driver/driver_configs.jl

```diff
@@ -339,6 +343,78 @@ function OceanBoxGCMConfiguration(
     )
 end
 
+function OceanSplitExplicitConfiguration(
+    name::String,
+    N::Int,
+    (Nˣ, Nʸ, Nᶻ)::NTuple{3, Int},
+    param_set::AbstractParameterSet,
+    model_3D::OceanModel,
+    model_2D::BarotropicModel;
+    FT = Float64,
+    array_type = ClimateMachine.array_type(),
+    solver_type = ExplicitSolverType(
+        # solver_method = SplitExplicitLSRK2nSolver,
```

Comment:
Assume you'll remove any commented lines of code before merging.

Reply:
Yeah, this was just to get the driver working. I probably need to add an entirely new solver type here. That's the second half of this PR.